etc/gnome-ring.profile

```diff
@@ -0,0 +1,32 @@
+# Firejail profile for gnome-ring
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/default.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+noblacklist ${HOME}/.local/share/gnome-ring
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+
```

Comment:
Does it work with `include /etc/firejail/whitelist-var-common.inc` ?

Reply:
Had to add /etc/firejail/whitelist-var-common.inc to my /etc/ folder since I didn't have it. It does seem to work.